patches/google-cloud-bigquery-JobWaitTrait.php.patch

```diff
@@ -1,12 +1,18 @@
 --- src/JobWaitTrait.php	2023-11-01 13:07:13.826212516 +0100
 +++ src/JobWaitTrait.php	2023-11-01 13:07:24.542742533 +0100
-@@ -55,7 +55,9 @@
+@@ -55,7 +55,15 @@
                  }
              };
 
 -            (new ExponentialBackoff($maxRetries))
 +            (new ExponentialBackoff($maxRetries, function (\Throwable $e) {
-+                return $e->getCode() !== 499;
++                if ($e->getCode() === 499) {
```

Comment:
Doplnil bych komentar ze tu 499 vraci query timeout jobu. Divny je ze to nezminuji ted v doc :)
![Uploading image.png…]()

Reply:
Přidal sem a dal sem obecně tu 400